nyc/people.py

```diff
@@ -64,49 +64,53 @@ def scrape(self):
                 else:
                     role = 'Council Member'
 
-                district = web.get('District', '').replace(' 0', ' ')
+                if web:
+                    district = web.get('District', '').replace(' 0', ' ')
+                else:
+                    district = ''
```

Comment:
Instead of the `else`...you could do:

```
district = ''
if web:
......
```

Reply:
I find the `if / else` construct easier to grok, and we seem to use it frequently elsewhere – is that alright?

Reply:
All right by me!